src/main/kotlin/com/autonomousapps/internal/kotlin/PublicApiDump.kt

```diff
@@ -57,8 +58,20 @@ fun getBinaryAPI(classStreams: Sequence<InputStream>, visibilityFilter: (String)
             it.isEffectivelyPublic(classAccess, mVisibility)
           }
 
-          ClassBinarySignature(name, superName, outerClassName, supertypes, memberSignatures, classAccess,
-              isEffectivelyPublic(mVisibility), metadata.isFileOrMultipartFacade() || isDefaultImpls(metadata)
+          val annotations = visibleAnnotations.orEmpty().map { it.desc.replace("/", ".") }
+
+          ClassBinarySignature(
```

Comment:
So, the reason I didn't format this file to project standards is that it's basically copy-pasted from JetBrains (as you can see from the top of the file). So I'm a little hesitant to make too many changes and lose the ability to update from the original. Admittedly an annoying situation.

I don't know a better way to do this, so I guess I just like talking to myself.

Reply:
Hmm, I could just leave a comment explaining it? I like region comments too to isolate fork changes

Reply:
a6ece6b